charts/victoria-metrics-cluster/CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 ## Next release
 
-- TODO
+- Remove unsupported `scheme` field under `livenessProbe.tcpSocket`. See [#844](https://github.com/VictoriaMetrics/helm-charts/pull/844) by @MisguidedEmails.
```

Comment:
Should this be included in this PR?

Reply:
> Should this be included in this PR?

yeah, [it](https://github.com/VictoriaMetrics/helm-charts/pull/845/commits/7ba4bf03f133c101f82632ef2f6293cf997d2d41) is a follow up for https://github.com/VictoriaMetrics/helm-charts/pull/844/commits/f997bd619be019d69be1c3c272b779c5649e6d2a.
Just pushed them in the same branch, I can create a new pr for this if it's feels wrong)